public/js/collections/albumReleases.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var _ = require('lodash');
+var Backbone = require('backbone');
+var AlbumReleases = require('../models/albumReleaseModel');
+
+module.exports = Backbone.Collection.extend({
+    model: albumReleases,
+    url: '/complete-library',
+    getLibrary: function () {
+      return model.get('artist');
+    }));
+  }
+});
```

Comment:
All the Backbone components need to be modules. This doesn't have to be CommonJS module format, but it needs to be a JavaScript module.